screencaptor/src/main/java/com/wealthfront/screencaptor/ViewMutation.kt

```diff
@@ -0,0 +1,12 @@
+package com.wealthfront.screencaptor
+
+import androidx.test.espresso.ViewAction
+import androidx.test.espresso.ViewInteraction
+
+interface ViewMutation {
+  fun getPerformInteraction(): ViewInteraction
+  fun getPerformAction(): ViewAction
+
+  fun getRestoreInteraction(): ViewInteraction
+  fun getRestoreAction(): ViewAction
```

Comment:
each mutation knows how to undo itself. This is achieved by placing temporary data into View tags